components/Player/Pages/Overview.jsx

```diff
@@ -24,7 +24,7 @@ const PlayerMatchesTable = createTable(
 );
 const PlayerHeroesTable = createTable(
   playerHeroes.getPlayerHeroesById,
-  (state, sortState, playerId) => (sortState ? sortPlayerHeroes(playerId)(state) : transformPlayerHeroesById(playerId)(state)),
+  (state, sortState, playerId) => (sortState ? sortPlayerHeroes(playerId, 20)(state) : transformPlayerHeroesById(playerId, 20)(state)),
```

Comment:
shouldn't sorting occur on the full data, not the limited data?  (e.g., if we have 100 rows but only display 20, we should sort on the full data set and render the top 20 rows, not just sort on the displayed 20 rows).